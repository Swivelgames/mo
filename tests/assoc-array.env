declare -A repo
repo[resque]="Resque"
repo[hub]="Hub"
repo[rip]="Rip"
