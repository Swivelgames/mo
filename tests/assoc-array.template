{{#repo}}
  <b>{{@key}} - {{.}}</b>
{{/repo}}
