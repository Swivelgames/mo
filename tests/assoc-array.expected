  <b>hub - Hub</b>
  <b>rip - Rip</b>
  <b>resque - Resque</b>
